programs/sample_http_link/README.md

```diff
@@ -0,0 +1,49 @@
+#Running Sample HTTP Link
+##Setup Identities
+###Create Identities
+We will create a host and client identity.
+
+For the purposes of this example and for the purpose of brevity we will create the device identities and configure the endpoints all in one command though this can be done separately.
```

Comment:
imo... don't be brief... explain what each command does. someone looking for 'just the commands' (like i might) will just copy the steps that need to be done and skip all the words 

Also it'd be nice to consider a comprehensive idea here where we we simply state that you will need two identities. Use them or create them - here we are creating them. if you reuse identities make sure they have the proper attributes on them